packages/nuejs.org/docs/settings.md

```diff
@@ -37,46 +37,46 @@ links:
 
 
 ### lightning_css
-Setting this to `false` disables the default processing with [Lightning CSS](//lightningcss.dev/) and the CSS is served directly as is.
+Setting this to `false` disables the default processing by [Lightning CSS](//lightningcss.dev/) and the CSS is served directly as is.
 
 ### native_css_nesting
-Use native CSS nesting intead of converting them to un-nested style rules that are supported in all browsers. Setting this to `true` generates a smaller CSS output, but is not supported by the oldest of browsers. Check the current [Can I Use](//caniuse.com/css-nesting) statistics for details.
+Use native CSS nesting intead of converting them to un-nested style rules that are supported in all browsers. Setting this to `true` generates a smaller CSS output, but is not supported by older browsers. Check the current [Can I Use](//caniuse.com/css-nesting) statistics for details.
 
 ### port
-The port number of the development server. Default is 8080
+The port number of the development server. Default is 8080.
 
 ### syntax_highlight
 Set this to `false` to disable the [built-in stylesheet](syntax-highlighting.html) for code blocks.
 
 ### view_transitions
-Setting this to `true` enables [view transitions](reactivity.html#view-transitions) for instant and smooth page switches
+Setting this to `true` enables [view transitions](reactivity.html#view-transitions) for instant and smooth page switches.
 
 
 
 
 
 ## Site settings
-List of site-wide settings that impact your SEO data and other metadata inside your document `<head>` element. These are defined on your `site.yaml` but can be overwritten on area- and page level.
+List of site-wide settings that impact your SEO data and other metadata inside your document `<head>` element. These are defined in your `site.yaml` but can be overwritten on area and page level.
 
 
 ### base
-The value for HTML [<base>](//developer.mozilla.org/en-US/docs/Web/HTML/Element/base) attribute
+The value of HTML [<base>](//developer.mozilla.org/en-US/docs/Web/HTML/Element/base) attribute.
 
 ### charset
 The value of the charset meta tag. The default is "utf-8".
 
 ### direction
-The value of `<html direction="{ direction }">` attribute. The default is "ltr" (left to right)
+The value of `<html direction="{ direction }">` attribute. The default is "ltr" (left to right).
 
 ### favicon
 Relative path to your favicon that overrides the "favicon.ico" on the browser tab.
 
 ### generator
-The generator meta tag. The default is "Nue (nuejs.org)" and is only rendered for the production pages.
+The generator meta tag. The default is "Nue (nuejs.org)" and is only rendered on the production pages.
```

Comment:
```suggestion
The generator meta tag. The default is "Nue (nuejs.org)" and is only rendered on production pages.
```
(I think `for` fits best, but I understand, that `on` emphasizes, that it's in the page's html)

Or maybe just `in production (mode)`?

Reply:
I think in production (mode) seems the best choice.

Reply:
Okay. 
Do you like `in production mode.` or `in production.` better, here?

Reply:
I think in production mode makes it clearer what it is.